CHANGELOG.md

```diff
@@ -1,660 +1,753 @@
-- Remove the distinction between Fund and Activity from the user
-- Users can edit an organisation
-- Users can edit the basic fund record
-- Users can edit a transaction
-- Users can edit an activity record
-- Transactions record the provider and receiver organisations
-- All forms now use `govuk_design_system_formbuilder` instead of `simple_form`
-- Activity multi-step form now has validations
-- Users can view an XML representation of Transactions and Funds
-- Force SSL in production environments
-- A users role can be viewed, set and changed
-- Restrict delivery partners so they can only view and edit their own organisation
-- Fund managers can manage users, organisations, funds, fund activites and fund transactions
-- Transaction and Activity dates are restricted to 10 years in the past or 25 years in the future at most
-- Provide a way to flag an organisation as BEIS
-- User email addresses must be valid emails
-- Users are only associated with one organisation
-- Users land on their organisation#show page when they log in, instead of a "dashboard"
-- Fund manager can add a programme level activity to a fund level activity
-- Fund manager can view a fund level activity's programme activities
-- Fund managers can create Budgets
-- Fund and Programme activities store funding organisation details
-- Fund and Programme activities store accountable organisation details
-- Fund activities store extending organisation details
-- Sign in button works when JS is disabled
-- Fund managers can set and change the extending organisation
-- Date inputs in forms for creating activity, transaction and a budget have a hint text
-- Transaction provider and receiver IATI references are exposed in the XML if present
-- Users can report project level activities
-- Users can add budgets to project level activities
-- Users can view Budgets in the Activity XML
-- Consolidate all user roles into 'administrator', removing 'fund_manager' and 'delivery_partner'. All users can temporarily do anything
-- Users who belong to delivery partners see a list of programmes on their organisation page
-- Users are authorised based on the organisations they belong to
-- Users can add budgets to activities at all levels
-- Users can edit a budget
-- Users can add currency information to budgets and see the currency in the budget XML
-- Users can report an activities implementing organisation
-- Users can edit a reported implementing organisation
-- Users can view implementing organisations in the Activity XML
-- Planned start and end dates are mandatory
-- Actual start and end dates must not be in the future
+[Full changelog][unreleased]
+
+- Add budget revisions:
+  - Only budget value can be edited
+  - Add a budget revisions page
+  - Add comments when editing budgets to be shown alongside the revisions
+  - Improve design/content for the budgets area
+  - Show "original" and "revised" budgets on the XML exports
+
+## 132 / 2023-03-16
```

Comment:
I'd prefer to see "Release" here, either as "release-132" or "Release 132" if we think that's neater. It _should_ be obvious what the numbers refer to here but that might not always be the case, particularly if this is a repository that's open to the public. I also find it a little easier to scan 👀

Reply:
I did wonder about putting them all under a "Released" `h2` but that felt a little off. Maybe this?

Any thoughts @jkempster34? I wasn't sure if your comment on titles actually included the shifting of the changelog link actually, or just the format (Keep a Changelog includes the link in the heading)

---

## Release 132 - 2023-03-16

[Full changelog](example.com)

Reply:
Um, honestly I just didn't like the slash because it looked funny to me!

I don't have any preferences beyond that